### PR TITLE
Update CommandServer.java

### PR DESCRIPTION
**Pull request by ZeProf2Code**

### CODE REVIEW

**Comment by md-5** [comment]
Yeah uh that's now what I meant.

It's new TextComponent( part that should be replaced by legacy text

**Comment by Janmm14** [comment]
*not

File: module/cmd-server/src/main/java/net/md_5/bungee/module/cmd/server/CommandServer.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ public void execute(CommandSender sender, String[] args)
             }
 
             TextComponent serverList = new TextComponent( ProxyServer.getInstance().getTranslation( "server_list" ) );
-            serverList.setColor( ChatColor.GOLD );
+            serverList.setColor( TextComponent.fromLegacyText() );
```
Comment (Janmm14) [inline module/cmd-server/src/main/java/net/md_5/bungee/module/cmd/server/CommandServer.java]: This code cannot compile. Learn java?

**Comment by md-5** [comment]
Gold is no longer forced

**Pull request closed without merging**